services/server/test/chains/chain-tests.spec.ts

```diff
@@ -67,18 +76,6 @@ describe("Test Supported Chains", function () {
     }
   });
 
-  // log server response when test fails
-  afterEach(function () {
-    const errorBody = currentResponse && currentResponse.body;
-    if (this.currentTest.state === "failed" && errorBody) {
-      console.log(
-        "Server response of failed test " + this.currentTest.title + ":"
-      );
-      console.log(errorBody);
-    }
-    currentResponse = null;
-  });
-
```

Comment:
Why are we removing this? This was helpful in seeing the server's error response. Or did we replace it? Otherwise mocha only gives `Expected status: 200 Actual: 500`

Reply:
I removed it because the `currentResponse` variable was never set. It was commented in all places. I can bring it back but then we should also uncomment where the variable is assigned.

Reply:
Hmm ok just remove it then. Now that we have proper logging, this is probably not needed